Src/VimCore/EditorUtil.fs

```diff
@@ -1782,6 +1782,43 @@ module TextViewUtil =
         let caret = GetCaret textView
         caret.EnsureVisible()
 
+    /// Check whether we should recenter the view on point
+    let CheckRecenterOnPoint (textView : ITextView) point =
+
+        // Be careful because using ITextViewLines can result in an exception.

```

Comment:
When can this exception happen?

Reply:
I did not personally experience these exceptions. I added the try block because I saw similar code elsewhere in VsVim, for example in `CommandUtil.ScrollLines`:

```fsharp
        try
...
        with
        // Dealing with ITextViewLines can lead to an exception (particularly during layout).  Need
        // to be safe here and handle the exception.  If we can't access the ITextViewLines there
        // isn't much that can be done for scrolling
        | _ -> ()
```
The documentation suggests certain methods can throw `InvalidOperationException`, for example `GetTextViewLineContainingBufferPosition` for the reason:

> The view has not completed initialization.

Rather than, say, "during layout." It seems wise to prevent these exceptions, should they occur, especially since what we are doing is entirely cosmetic.